providers/summary/scancode.js

```diff
@@ -153,9 +154,16 @@ class ScanCodeSummarizer {
     return result.length === 0 ? null : result
   }
 
-  _collectAttributions(copyrights, holders) {
+  _collectAttributions(copyrights) {
     if (!copyrights || !copyrights.length) return null
-    return this._setToArray(copyrights.reduce((result, entry) => this._addArrayToSet(entry.holders, result), new Set()))
+    const set = new Set()
+    for (let copyright of copyrights) {
+      if (!copyright.statements) continue
+      const statements = copyright.statements.map(statement => he.decode(statement)
+        .replace(/\\[nr]/, '\n'))
```

Comment:
suspect we should just remove these. that is, each statement should be one line. Actually, in that case, we should also remove real CR and LF instances so the statement is on one line. Perhaps replace with a space and then overall remove multiple whiltespace.

Reply:
Done.